README.md

```diff
@@ -1,2 +1,99 @@
-# website-template
-A template to create all public facing sites
+# Tiny Site Backend Setup Guide
+
+This README provides step-by-step instructions for setting up the Tiny Site Backend on your local development environment. The Tiny Site Backend is built using the Go programming language, PostgreSQL for the database, and Google authentication for user management. Before you begin, ensure that you have the following prerequisites installed on your system:
+
+1. Go 1.21
+2. PostgreSQL
+3. pgAdmin (optional but recommended for database management)
+
+## Prerequisites
+
+### Go 1.21
+
+Make sure you have Go version 1.21 or higher installed on your system. You can download and install Go from the official website: [Go Downloads](https://golang.org/dl/)
+
+### PostgreSQL
+
+You will need PostgreSQL as the database for the Tiny Site Backend. You can download and install PostgreSQL from the official website: [PostgreSQL Downloads](https://www.postgresql.org/download/)
+
+### pgAdmin (Optional)
+
+pgAdmin is a popular database management tool for PostgreSQL. It is recommended to install pgAdmin for easier database management. You can download and install pgAdmin from the official website: [pgAdmin Downloads](https://www.pgadmin.org/download/)
+
+## Setup Instructions
+
+1. Clone the Tiny Site Backend repository from GitHub:
+
+   ```bash
+   git clone https://github.com/Real-Dev-Squad/tiny-site-backend.git
+   ```
+
+2. Navigate to the project directory:
+
+   ```bash
+   cd tiny-site-backend
+   ```
+
+3. Create a `.env` file for your local development environment. You can use the sample `.env` file provided in the `environments` directory as a template. Copy the `dev.env` file to `.env`:
+
+   ```bash
+   cp environments/dev.env .env
+   ```
+
+   Edit the `.env` file and configure the environment variables according to your local setup. Make sure to set the database connection details and Google OAuth credentials.
```

Comment:
> Make sure to set the database connection details and Google OAuth credentials.

@sahsisunny Does you explained above point how to set?

Reply:
Have you seen the dev.env?